handlers/misttriggers/recording_end.go

```diff
@@ -143,6 +143,11 @@ func (d *MistCallbackHandlersCollection) triggerRecordingEndSegmenting(w http.Re
 		// prepare .dtsh headers for all rendition playlists
 		for _, output := range outputs {
 			// output is multivariant playlist
+			err := createDtsh(requestID, output.Manifest)
+			if err != nil {
+				// should not block the ingestion flow or make it fail on error.
+				log.Log(requestID, "master createDtsh() failed", "code", err, "destination", output.Manifest)
```

Comment:
You can use the LogError method, which makes sure our err field always has the same key